src/SDKs/StorSimple8000Series/StorSimple8000Series.Tests/StorSimple8000Series.Tests.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.Compute" Version="15.0.0" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="10.1.0-preview" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure" Version="3.3.8" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="1.6.0" />
+    <PackageReference Include="xunit" Version="2.2.0" />
```

Comment:
@manaas-microsoft can you remove *.Test.Sdk, ClientRuntime.Azure, *.TestFramewok, and all xunit package references.
And add <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" /> as one of the imports on line 2
You can model your test project looking at other test projects in the repo.